aws_signing_helper/serve.go

```diff
@@ -158,6 +158,7 @@ func AllIssuesHandlers(cred *RefreshableCred, roleName string, opts *Credentials
 		expirationTime := time.Now().Add(time.Second * time.Duration(tokenTTL))
 		InsertToken(token, expirationTime)
 
+        w.Header().Set(EC2_METADATA_TOKEN_HEADER, tokenTTLStr)
```

Comment:
Stay consistent w/ whitespace.

Reply:
Thanks for the catch - I've fixed it in my next revision.